### PR TITLE
Removing const / let in favor of var, for Chrome 45. See #2098.

### PR DESCRIPTION
**Pull request by joeldenning**
See #2098

### CODE REVIEW

File: src/features/node-fetch.js [hunk 1]
```diff
@@ -6,12 +6,12 @@ import { fileURLToPath } from 'url';
 import rimraf from 'rimraf';
```
Comment (guybedford) [inline src/features/node-fetch.js:6]: No need to do it for this file.

File: src/system-node.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import './extras/module-types.js';
 import './features/node-fetch.js';
 import { BASE_URL } from './common.js';
 
-export const System = global.System;
+export var System = global.System;
```
Comment (guybedford) [inline src/system-node.js:10]: Or for this one.

**Comment by joeldenning** [comment]
The tests are failing for this - it appears the named-exports extra was relying on the semantics of const/let in a way that doesn't work with var. I am working on updating it.

**Comment by joeldenning** [comment]
The tests are failing on the master branch, too. I will submit a subsequent PR to fix the tests.

**Pull request merged**